applications/minotari_console_wallet/src/grpc/wallet_grpc_server.rs

```diff
@@ -1175,6 +1185,199 @@ impl wallet_server::Wallet for WalletGrpcServer {
         Ok(Response::new(TransferResponse { results }))
     }
 
+    #[allow(clippy::too_many_lines)]
+    async fn range_limited_coin_join(
+        &self,
+        request: Request<RangeLimitedCoinJoinRequest>,
+    ) -> Result<Response<TransferResponse>, Status> {
+        let message = request.into_inner();
+        debug!(target: LOG_TARGET, "range_limit_coin_join: {:?}", message);
+
+        // Simple verification of range and target amount
+        let range = message.lower_bound..message.upper_bound;
+        if message.lower_bound * message.upper_bound == 0 || message.lower_bound >= message.upper_bound {
```

Comment:
```suggestion
        if message.lower_bound >= message.upper_bound {
```
This is weird, check that forces certain behaviour, why not allow the user to specify 0 as the lower bound, which will include the smallerst utxo's in the wallet. 
This also removes the overflow check which is not safely handled